<launch>

    <!-- Larger smoothing factor means more smoothing 
        while smaller values indicate less smoothing. -->
    <!-- https://scipy.github.io/devdocs/generated/scipy.interpolate.splprep.html#scipy.interpolate.splprep -->

    <!-- Configurable arguments -->
	<arg name="spline_degree" default="3" />
    <arg name="smoothing_factor" default="0.1" />
	<arg name="waypoint_spacing" default="0.1" />

	<node name="bspline" pkg="trajectories" type="bspline.py">
		<param name="spline_degree" value="$(arg spline_degree)" type="int" />
        <param name="smoothing_factor" value="$(arg smoothing_factor)" type="double" />
		<param name="waypoint_spacing" value="$(arg waypoint_spacing)" type="double" />
	</node>

</launch>
